client/src/components/mobile/NavigationMobile.js

```diff
@@ -11,7 +11,7 @@ const NavMobile = (props: Props) => {
   const { authed } = props;
 
   return (
-    <nav role="navigation">
+    <nav id="nav" role="navigation">
```

Comment:
Thank you for doing this :) should we make it `nav-mobile` just to prevent class confusion in the future (like if we're inspecting element in a browser).

Reply:
Yep, that sounds good